src/pages/management/MachinePage.tsx

```
@@ -1,94 +1,124 @@
+
   useEffect(() => {
     if (isModalOpen === true) return;
     const getTableData = async () => {
-      const res = await getSearchCompanyDevice();
+      const res = await getSearchCompanyDevice({
+        page: currentPage,
+        size: currentSize,
+      });
       if (res.pass) {
-        setTableData(res.data);
+        setTableData((res.data as any).content);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)